data/README.md

```diff
@@ -0,0 +1 @@
+test data goes here
```

Comment:
Do we need a test data directory? I believe that this is better tracked in the `${module_name}/src/test/resources` unless it is share by several modules.

A proper way of sharing the resources and test helpers is to create another module `htsjdk-test`; this also will allow to include test dependencies in the artifact (e.g., JUnit/TestNG dependencies, or HDFS/GCS to test in other filesystems).

Reply:
So I think the htsjdk/gatk project structure of having test/resources directory that includes all the test data files was probably a mistake.  I believe the test resources is really meant to be for resources that should be packaged into the jar for access at test runtime. This includes things like `log4j.xml` files and configuration files for mocks, etc.  

It takes a significant amount of time to package test data into the jar, and we don't actually want it there, since we want to be referring to it by file path.  In gatk we have modifications to the test packaging commands that strips out all of the test *data* files when packaging the test jar, but leaves in the resource files.  It's crufty and causes confusion, so I would rather just have them separated.

Reply:
I think that to some extend makes sense to package the data in the test resources to be able to distribute the packaged jar with all the content - also true that we should be more careful how we access the data in that case.

Another option is to set an integration-test separate for the unit test, for data-related testing (see, for example, https://www.petrikainulainen.net/programming/gradle/getting-started-with-gradle-integration-testing/)

Reply:
Let's open another issue to decide this...

Reply:
#19